homework/27-28/week-03/README.md

```diff
@@ -6,39 +6,46 @@ If you have any remaining time, please complete/review the exercises from dawn a
 
 Please refer to the [Instructions for Submitting Homework](/how-to/homework-submission.md).
 
-<!--
+
 ## Day 1
 
-1. Reading
-    * [Example](#example)
-2. Bonus/Stretch
-    * [Example](#example)
+### REST API homework
+
+1. Read [the top comment in this discussion](http://stackoverflow.com/questions/671118/what-exactly-is-restful-programming) about what REST is (and the rest of the discussion if you're curious).
```

Comment:
Maybe a note not to feel like they need to understand everything in this post. For instance, we don't usually go over HATEOS in WDI